packages/react/src/organisms/sectionHeader/base.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactNode } from "react";
+import "../../../tailwind.css";
+import { useThemeContext } from "../../theme/ThemeProvider";
+import resolvedStyleProps from "../../utils/resolvedStyleProps";
+import classNames from "../../utils/classNames";
+
+export interface SectionHeaderBaseProps {
+  textAlign?: string;
+  title?: any;
+  subTitle?: any;
+  actions?: any;
+  horizontalPadding?: string;
+  verticalPadding?: string;
+  backgroundClasses?: string;
+  widthClasses?: string;
+}
+
+const SectionHeaderBase = (props: SectionHeaderBaseProps) => {
+  const backgroundClasses = props.backgroundClasses || "bg-gray-50";
+  const widthClasses =
```

Comment:
@ShubhamJoshi-1994 widthClasses and backgroundClasses should come from theme.

Just think this way, a developer might pass overriding props everytime they use SectionHeaderBase, or they might just create a variant one place and pass "variant" prop. Variant prop is more flexible, that's why styles should be resolved using resolveProps and shouldn't be passed like above.

Reply:
This file is removed, removed hard coded classes from other components, using theme file.